src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/WXOWXOP/XamlCodeGenerator_Binding_Xaml_Object_With_Xaml_Object_Properties_5147419e44d1bc3e3f86860ad528476f.cs

```diff
@@ -100,6 +100,7 @@ private void InitializeComponent()
 			))
 			;
 			
+			
```

Comment:
Best to avoid this extra line if possible.
Also best to have the test as a Xaml gen test instead of a dummy XAML file.

Reply:
It should be testable for sure. You almost have complete control of how the compilation is created with the existing test infrastructure.

I think you need to add `Resources.resw` as `AdditionalFile` with `SourceItemGroup="PRIResource"` (the same as how the tests add xaml files as AdditionalFiles with `SourceItemGroup="Page"`